birthdays/application.py

```diff
@@ -0,0 +1,14 @@
+import os
+
+from cs50 import SQL
+from flask import Flask, flash, jsonify, redirect, render_template, request, session
+
+app = Flask(__name__)
+db = SQL("sqlite:///birthdays.db")
+
+
+@app.route("/", methods=["GET", "POST"])
+def index():
+    # TODO: If the request method is GET, display the entries in the database. If the method is POST, add the entry into the database.
```

Comment:
Just to help students along here, I think it's probably reasonable for us to include the `if`/`else` check for `request.method` in the distribution code